types/transform-pouch/tsconfig.json

```
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
```

```suggestion
            "es6"
```
why it needs to depend on DOM library?

---

Hi, when removing the dom, the test fail because the parent type needs it.

Error: Errors in typescript@4.3 for external dependencies:

../pouchdb-core/index.d.ts(93,19): error TS2304: Cannot find name 'Request'.
../pouchdb-core/index.d.ts(94,12): error TS2304: Cannot find name 'RequestInit'.
../pouchdb-core/index.d.ts(95,14): error TS2304: Cannot find name 'Response'.

---

what about:
```diff
diff --git a/types/transform-pouch/transform-pouch-tests.ts b/types/transform-pouch/transform-pouch-tests.ts
index 9c056ae414..b61fbf2857 100644
--- a/types/transform-pouch/transform-pouch-tests.ts
+++ b/types/transform-pouch/transform-pouch-tests.ts
@@ -1,3 +1,5 @@
+/// <reference lib="dom" />
+
 import * as PouchDB from "pouchdb";
 import * as transform from "transform-pouch";
 PouchDB.plugin(transform);
diff --git a/types/transform-pouch/tsconfig.json b/types/transform-pouch/tsconfig.json
index 3e1f2589e5..f2349342b6 100644
--- a/types/transform-pouch/tsconfig.json
+++ b/types/transform-pouch/tsconfig.json
@@ -2,8 +2,7 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6",
-            "dom"
+            "es6"
         ],
         "noImplicitAny": true,
```
this is up to you. There is no support for skip-lib-check on DT. Moving that dependency into test,
removes direct dependency from your package details. I saw that other `couch-*` depends on `dom` also in their tsconfigs.